### PR TITLE
feat: add stats for account

### PR DESCRIPTION
**Pull request by tommie1001**
go to /stats/account/{account_id} and see the stats!

### CODE REVIEW

**Comment by JaZo** [comment]
:warning: The following controllers contain actions outside of the 7 resource actions (`index`, `create`, `store`, `show`, `edit`, `update`, `destroy`). For more details, review [the docs](https://laravel.com/docs/controllers#resource-controllers) or watch [Cruddy by Design](https://www.youtube.com/watch?v=MF0jFKvS4SI) to see if you may rework these into *resource controllers*.

- [ ] app/Http/Controllers/StatsController.php

**Comment by JaZo** [comment]
:information_source: Now with type hints in your code, defining types within PHP DocBlocks is redundant. Laravel has removed all of the `@param` and `@return` tags from its DocBlocks where types are defined with PHP. Similarly, Shift [removed these tags](https://github.com/swisnl/dlf-hackathon-2023/pull/34/commits/4e0df038909c5b196fbf0b93df4d475a7dc02835?w=1) from any DocBlock where the code now has _equivalent_ type hints.

**Comment by JaZo** [comment]
:information_source: While the _Laravel Fixer_ performs automation to adopt the latest Laravel conventions, Shift understands developers have different preferences. All of Shift's automation is done in nice, atomic commits. This makes it easier to undo any of the changes Shift makes using `git revert`.

File: app/Http/Controllers/StatsController.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+use App\Models\Account;
+use App\Models\Tenant;
+use Illuminate\Http\Response;
+
+class StatsController extends Controller
+{
+    public function showTenant(Tenant $tenant): Response
+    {
+        $transactions = $tenant->transactions->where('mollie_status', 'paid');
```
Comment (Rocksheep) [inline app/Http/Controllers/StatsController.php]: Can you add a get behind this?

File: app/Http/Controllers/StatsController.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            view('stats.view', [
+                'centsCharged' => $transactions->pluck('cents_charged')->sum(),
+                'gramsOfCo2' => $transactions->pluck('grams_of_co2')->sum(),
+            ])->render()
+        )->header('Content-Type', 'image/svg+xml');
+    }
+
+    public function showAccount(Account $account): Response
+    {
+        $transactions = $account->transactions->where('mollie_status', 'paid');
```
Comment (Rocksheep) [inline app/Http/Controllers/StatsController.php]: Same here

File: app/Http/Controllers/StatsController.php (hunk 1 above)
Comment (Rocksheep) [reply app/Http/Controllers/StatsController.php]: Can you also use the paid status from the PaymentStatus?

**Review by Rocksheep — CHANGES_REQUESTED** [review]

**Review by Rocksheep — APPROVED** [review]
LGTM

**Pull request merged**